### PR TITLE
Update chip-tool documentation.

### PR DESCRIPTION
**Pull request by bzbarsky-apple**
The pairing bits were quite out of date

#### Problem
Docs out of date

#### Change overview
Use the right pairing command format, update the build instructions.

#### Testing
I wish we had automated testing that the docs match reality.

### CODE REVIEW

**Review by saurabhst — APPROVED** [review]

File: examples/chip-tool/README.md [hunk 1]
```diff
@@ -12,48 +12,65 @@ An example application that uses CHIP to send messages to a CHIP server.
...
-the ESP32 all-clusters-app:
+the ESP32 all-clusters-app to commission it onto a Wi-Fi network:
+
+    $ chip-tool pairing ble-wifi ssid password 0 20202021 3840
+
+where:
+
+* ssid is the Wi-Fi SSID either as a string, or in the form hex:XXXXXXXX where the bytes of the SSID are encoded as two-digit hex numbers.
+* paswword is the Wi-Fi password, again either as a string or as hex data
+* The 0 is the fabric id, until more compelete support for multiple fabrics is implemented in our commissioning process.
```
Comment (carol-apple) [inline examples/chip-tool/README.md:63]: nit: typo `compelete`

Comment (bzbarsky-apple) [reply examples/chip-tool/README.md]: Good catch, fixed.

File: examples/chip-tool/README.md [hunk 2]
```diff
@@ -12,48 +12,68 @@ An example application that uses CHIP to send messages to a CHIP server.
...
 
 Run the built executable and pass it the discriminator and pairing code of the
-remote device.
+remote device, as well as the network credentials to use.
 
 The command below uses the default values hard-coded into the debug versions of
-the ESP32 all-clusters-app:
+the ESP32 all-clusters-app to commission it onto a Wi-Fi network:
+
+    $ chip-tool pairing ble-wifi ssid password 0 20202021 3840
```
Comment (jmeg-sfy) [inline examples/chip-tool/README.md:58]: following your PR , then someone must change the internal wording of chip-tool itself to commission

**Review by jmeg-sfy — APPROVED** [review]

**Review by andy31415 — APPROVED** [review]

File: examples/chip-tool/README.md (hunk 2 above)
Comment (bzbarsky-apple) [reply examples/chip-tool/README.md:58]: I think that would be a good idea, yes, but didn't want to scope-creep this PR...

**Review by woody-apple — APPROVED** [review]

**Pull request merged**